rotkehlchen/inquirer.py

```diff
@@ -558,18 +559,31 @@ def find_curve_pool_price(
         """
         assert self._ethereum is not None, 'Inquirer ethereum manager should have been initialized'  # noqa: E501
 
-        pools = get_curve_pools()
-        if lp_token.evm_address not in pools:
+        pool_addr_in_cache = GlobalDBHandler().get_general_cache_value(
+            cache_type=GeneralCacheType.CURVE_POOL_ADDR,
+            name=lp_token.evm_address,
+        )
+        if pool_addr_in_cache is None:
+            return None
+        pool_addr = string_to_evm_address(pool_addr_in_cache)
```

Comment:
comment here that the cache value is guaranteed to be a checksummed evm address due to the way we save it in the DB